schemata/export_files.json

```diff
@@ -0,0 +1,115 @@
+      "method": "POST",
+      "rel": "create",
+      "title": "",
+      "http_header": { "x-api-key": "0123456789abcdef0123456789abcdef" },
+      "targetSchema": {
+        "properties": {
+          "href": {
+            "type": "string",
+            "description": "Hypertext reference to this resource.",
+            "pattern": "/api/v1/user_identities/\\d+/exports/\\d+/export_files",
+            "example": "/api/v1/user_identities/37442/exports/56791/export_files"
+          },
+          "export_files": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "required": ["id", "href", "export_id", "status"],
+              "properties": {
+                "id": {
+                  "type": "integer",
+                  "example": 74780,
+                  "description": "Unique identifier of this export file."
+                },
+                "href": {
+                  "type": "string",
+                  "description": "Hypertext reference to this resource.",
+                  "pattern": "/api/v1/exports/\\d+/export_files/\\d+",
+                  "example": "/api/v1/exports/86466/export_files/74780"
+                },
+                "export_id": {
+                  "type": "integer",
+                  "example": 86446,
+                  "description": "Unique identifier of the associated export."
+                },
+                "status": {
+                  "type": "string",
+                  "description": "Current status of this export file.",
+                  "enum": ["initializing", "queued", "in_progress", "available", "success_with_errors", "empty_list", "failed"],
+                  "example": "available"
+                }
+              }
+            }
+          }
+        }
+      },
+      "schema": {
+        "type": "object",
+        "properties": {
+          "callback": {
+            "type": "string",
+            "format": "uri",
+            "description": "For requests using the optional “callback” parameter, you should build your request as follows: /api/v1/user_identities/:user_identity_id/exports/:export_id/ export_files?callback=http://someurl.com/your_callback.cgi When the file is ready, WebAdMIT will POST to the callback URL that you specify with the following form POST parameters: export_file[id] export_file[export_id] export_file[status] An example of the body of a POST to a callback when a file is ready is as follows: export_file[id]=99752&export_file[export_id]=123316&export_file[status]=Available This gives you the export_file_id of the file that’s ready for download once it is available (you will use the export_id and the export_file_id POSTed here to form the next API call in item 5 below to actually get the URL to download the now-ready file). We recommend the use of the public RequestBin (http://requestb.in/) service in order to test out how this particular call works.",
+            "pattern": "^http://",
```

Comment:
I think we should just remove this pattern... I don't really see a value in enforcing `http` or `https` when `www.example.com` will work fine